build.sbt

```diff
@@ -30,7 +30,11 @@ val filename = "src/main/resources/docs/patterns.json"
 
 val toolMap = JSON.parseFull(Source.fromFile(filename).getLines().mkString).get.asInstanceOf[Map[String,String]]
 
-val rubocopVersion = toolMap("version")
+val defaultVersion = "0.49.0"
+
+val rubocopVersion = { val jsonVersion = toolMap("version")
+  if(jsonVersion.isEmpty) defaultVersion else jsonVersion
```

Comment:
when you have an option, the most idiomatic way to provide a default value would be by using `.map(...).getOrElse(...)` 

`val rubocopVersion = toolMap("version").getOrElse("0.49.0")`